### ISSUE TITLE
Scrollbar doesn't work while dragging over gutter of sibling split pane

### ISSUE THREAD

**Issue by phisch** [zed-industries/zed#11742]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When grabbing the scroll bar, and dragging it over the gutter of a sibling split pane, it stops updating its scroll position.

https://github.com/zed-industries/zed/assets/1282767/597877b2-b853-4fbc-bac6-55cd2b600961

There was a very similar bug that was solved in #10737 which stopped other drags from updating while hovering the gutter, but the solution to that bug doesn't also fix this one, since editor scroll drags aren't "registered" as app drags (as far as I can understand from the code).

I couldn't quite find what causes the issue in this case. Do you have an idea @as-cii?

### Environment

Zed: v1.0.0 (Zed Dev c90263d59b8cde9e007f16b286f040fa849ecb24)
OS: Linux 1.0.0
Memory: 31.1 GiB
Architecture: x86_64

**Comment by MrSubidubi** [comment]
I think this was fixed recently via https://github.com/zed-industries/zed/pull/28924, would you care to check?

**Comment by phisch** [comment]
@MrSubidubi You are right. This resolves it. It's actually what I did in my initial PR for a similar issue about a year ago, but we decided on a different solution because we only looked at tab dragging: https://github.com/zed-industries/zed/pull/10737

Anyway, closing this, since it is resolved now.
